exec.go

```diff
@@ -305,7 +305,7 @@ func cut(et *Text, t *Text, _ *Text, dosnarf bool, docut bool, _ string) {
 			if n > RBUFSIZE {
 				n = RBUFSIZE
 			}
-			t.file.b.Read(q0, r[:n])
+			t.oeb.f.b.Read(q0, r[:n])
```

Comment:
`b` here is a RuneArray. That's suppose to be an internal detail of `File` which is itself an internal detail of `ObservableEditableBuffer`. So this needs a wrapper function on `ObservableEditableBuffer`.

Reply:
What should be done when RuneArray is used outside of the context of a file, such as with the snarfbuf or warning buffer? Should I coerce these into an OEB?